File: src/main/java/com/procurement/access/model/dto/tender/Document.java
package com.procurement.access.model.dto.tender;

import lombok.Data;

import java.util.List;

@Data
public class Document {
    public String id;
    public String documentType;
    public String title;
    public String description;
    public String url;
    public String datePublished;
    public String dateModified;
    public String format;
    public String language;
    public List<String> relatedLots;
}

File: src/main/kotlin/com/procurement/access/model/dto/ocds/Document.kt
package com.procurement.access.model.dto.ocds

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude
import com.procurement.access.domain.model.enums.DocumentType

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Document @JsonCreator constructor(

    val id: String,

    val documentType: DocumentType,

    var title: String?,

    var description: String?,

    var relatedLots: MutableList<String>?
)
